jfrog-applications/ci-and-sdks/ci-integrations/maven-artifactory-plugin.md

```diff
@@ -118,13 +118,13 @@ However, the Maven Artifactory Plugin provides many other configurations:
 |:--------------------:|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `<deployProperties>` | Specifies properties you can attach to published artifacts. For example: prop-value.                                                                                                                                                                     |
 |   `<artifactory>`    | Specifies whether environment variables are published as part of BuildInfo metadata and which include or exclude patterns are applied when variables are collected                                                                                       |
-|    `<publisher>`     | Defines an Artifactory repository where build artifacts should be published using a combination of a `<contextUrl>` and `<repoKey>/<snapshotRepoKey>`. Build artifacts are deployed if the deploy goal is executed and only after all modules are built. |
+|    `<publisher>`     | Defines an Artifactory repository where build artifacts should be published using a combination of a `<contextUrl>` and `<repoKey>/<snapshotRepoKey>`. Build artifacts are deployed only after the deployment goal is executed and all modules are built. |
```

Comment:
The goal is called "deploy" and therefore it shouldn't be referenced as "the deployment goal".